ext/search_kit/CRM/Search/Upgrader.php

```diff
@@ -18,6 +18,7 @@ public function enable() {
       ->addValue('icon', 'crm-i fa-search-plus')
       ->addValue('has_separator', 2)
       ->addValue('weight', 99)
+      ->addValue('permission', 'administer CiviCRM')
```

Comment:
@totten @colemanw is 'administer CiviCRM data' (sub permission) better here?

Reply:
Fine with me.